pkg/proxy/metaproxier/meta_proxier.go

```diff
@@ -94,7 +94,7 @@ func (proxier *metaProxier) OnServiceSynced() {
 func (proxier *metaProxier) OnEndpointsAdd(endpoints *v1.Endpoints) {
 	ipFamily, err := endpointsIPFamily(endpoints)
 	if err != nil {
-		klog.V(4).Infof("failed to add endpoints %s/%s with error %v", endpoints.ObjectMeta.Namespace, endpoints.ObjectMeta.Name, err)
+		klog.V(4).ErrorS("failed to add endpoints", "endpoints", klog.KObj(&endpoints.ObjectMeta), "error", err)
```

Comment:
No `V()` needed on `ErrorS` - but now I  wonder why this is V(4) - does it trigger often?

Doesn't `ErrorS` take err as the first arg?

Reply:
It seems that didn't have an `err`, so I will use `nil` to be the first arg.😳